code/modules/mob/living/simple_animal/parrot.dm

```diff
@@ -902,7 +906,7 @@
 /mob/living/simple_animal/parrot/Poly/death(gibbed)
 	if(!memory_saved)
 		var/go_ghost = 0
-		if(rounds_survived == longest_survival || rounds_survived == longest_deathstreak)
+		if(rounds_survived == longest_survival || rounds_survived == longest_deathstreak || prob(0.666))

```

Comment:
As the parrots age and the records get higher the opportunities to see the ghost will only grow rarer and rarer.  prob(0.666) is pretty goddamn rare for a roll that usually only happens once every other round or so.